### DISCUSSION TITLE
Automatically set status of task based on checkbox

### DISCUSSION THREAD

**Discussion by Cano707** [typst/typst#2386] [Ideas]
Hi everyone,

Since I am using Typst mainly as my task tracker, I would really like the addition of automatically updating the task status based on the checkbox of the task. 

This would be possible for the status "To Do" and "Done".

It would truly make life a bit easier, if I wouldn't have to check the checkbox and additionally update the status.

**Comment by Dherse** [comment]
Is this a web app feature? Than you should open an issue on the web app issues repo https://github.com/typst/webapp-issues

**Comment by Cano707** [comment]
I am sorry. I have actually mixed the discussion forums of Anytype and Typst. Yeah, I don't know, I am losing my mind I guess.
